### ISSUE TITLE
Optimized Loss Functions for Object detection and Application on Nighttime Vehicle Detection. (arXiv:2011.05523v1 [cs.CV])

### ISSUE THREAD

**Issue by CoffeeKumazaki** [CoffeeKumazaki/arXiv#826]
Optimized Loss Functions for Object detection and Application on Nighttime Vehicle Detection. (arXiv:2011.05523v1 [cs.CV])<br>
https://ift.tt/3eRJLB7<br>
<p>Loss functions is a crucial factor than affecting the detection precision in object detection task. In this paper, we optimize both two loss functions for classification and localization simultaneously. Firstly, by multiplying an IoU-based coefficient by the standard cross entropy loss in classification loss function, the correlation between localization and classification is established. Compared to the existing studies, in which the correlation is only applied to improve the localization accuracy for positive samples, this paper utilizes the correlation to obtain the really hard negative samples and aims to decrease the misclassified rate for negative samples. Besides, a novel localization loss named MIoU is proposed by incorporating a Mahalanobis distance between predicted box and target box, which eliminate the gradients inconsistency problem in the DIoU loss, further improving the localization accuracy. Finally, sufficient experiments for nighttime vehicle detection have been done on two datasets. Our results show than when train with the proposed loss functions, the detection performance can be outstandingly improved. The source code and trained models are available at https://ift.tt/2JUAehb.</p>
<br>
<br>
via cs.CV updates on arXiv.org http://arxiv.org/<br>
